### ISSUE TITLE
Why set the train_loader's shuffle is False ?

### ISSUE THREAD

**Issue by yingfenging** [jik876/hifi-gan#77]
Hello, thanks for your great work. I have a question about the blow code, why set the "shuffle" False ? If set Fasle, the same iteration of every epoch will get the same audio .
```
    train_loader = DataLoader(trainset, num_workers=h.num_workers, shuffle=False,
                              sampler=train_sampler,
                              batch_size=h.batch_size,
                              pin_memory=True,
                              drop_last=True)
```

Thank you very much, and look foward to your reply.

**Comment by jik876** [comment]
Sorry for the late reply. Thanks for your interest.
MelDataset and DistributedSampler shuffle the data.
Please refer to lines 76 and 81.

**Comment by PussyCat0700** [comment]
In single gpu mode DistributedSampler should not be able to do shuffling, so I think it won't work for single GPU [here](https://github.com/jik876/hifi-gan/blob/4769534d45265d52a904b850da5a622601885777/train.py#L81C53-L81C63) when `h.num_gpus==1`?

> Sorry for the late reply. Thanks for your interest. MelDataset and DistributedSampler shuffle the data. Please refer to lines 76 and 81.
